### ISSUE TITLE
Enabling the different services such as Google, Email, AWS etc.

### ISSUE THREAD

**Issue by jd-alexander** [TeamNobi/Nobi#5]
I was looking at the [kobo - install script](https://github.com/kobotoolbox/kobo-install) and I saw that there are several settings that can be enabled for the app such as Google, Email, Backups and such forth. What do I need to provide to have these enabled and the right api keys passed/stored in the environment variables?

**Comment by awemulya** [comment]
@jd-alexander please provide crediantials for.
aws
email
(google for what ??)

**Comment by jd-alexander** [comment]
Okay I will provide these. Are backups enabled?

**Comment by awemulya** [comment]
Yeah backups are all set. But If we use s3 for media and awsdb for postgres we must do it differently.

**Comment by jd-alexander** [comment]
Okay that is fine. I will let the media go to the server. for the support email it should be hello@getnobi.com
